tests/integrations/realcluster/reboot_pd_test.go

```diff
@@ -14,67 +14,45 @@
 
 package realcluster
 
-import (
-	"context"
-	"os"
-	"os/exec"
-	"testing"
-
-	"github.com/pingcap/log"
-	"github.com/stretchr/testify/require"
-)
-
-func restartTiUP() {
-	log.Info("start to restart TiUP")
-	cmd := exec.Command("make", "deploy")
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
-	err := cmd.Run()
-	if err != nil {
-		panic(err)
-	}
-	log.Info("TiUP restart success")
-}
-
 // https://github.com/tikv/pd/issues/6467
-func TestReloadLabel(t *testing.T) {
-	re := require.New(t)
-	ctx := context.Background()
-
-	resp, err := pdHTTPCli.GetStores(ctx)
-	re.NoError(err)
-	re.NotEmpty(resp.Stores)
-	firstStore := resp.Stores[0]
-	// TiFlash labels will be ["engine": "tiflash"]
-	// So we need to merge the labels
-	storeLabels := map[string]string{
-		"zone": "zone1",
-	}
-	for _, label := range firstStore.Store.Labels {
-		storeLabels[label.Key] = label.Value
-	}
-	re.NoError(pdHTTPCli.SetStoreLabels(ctx, firstStore.Store.ID, storeLabels))
-	defer func() {
-		re.NoError(pdHTTPCli.DeleteStoreLabel(ctx, firstStore.Store.ID, "zone"))
-	}()
-
-	checkLabelsAreEqual := func() {
-		resp, err := pdHTTPCli.GetStore(ctx, uint64(firstStore.Store.ID))
-		re.NoError(err)
-
-		labelsMap := make(map[string]string)
-		for _, label := range resp.Store.Labels {
-			re.NotNil(label)
-			labelsMap[label.Key] = label.Value
-		}
-
-		for key, value := range storeLabels {
-			re.Equal(value, labelsMap[key])
-		}
-	}
-	// Check the label is set
-	checkLabelsAreEqual()
-	// Restart TiUP to reload the label
-	restartTiUP()
-	checkLabelsAreEqual()
-}
+// func TestReloadLabel(t *testing.T) {
```

Comment:
these cases will be updated with new style in the next pr

Reply:
what do you think? Do I need update them in this pr? @rleungx

Reply:
Both are ok to me.